File: app/src/main/java/info/nightscout/androidaps/plugins/PumpDanaR/comm/MsgHistoryBolus.java
package info.nightscout.androidaps.plugins.PumpDanaR.comm;

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

/**
 * Created by mike on 20.07.2016.
 */
public class MsgHistoryBolus extends MsgHistoryAll {
    private static Logger log = LoggerFactory.getLogger(MsgHistoryBolus.class);
    public MsgHistoryBolus() {
        SetCommand(0x3101);
    }
    // Handle message taken from MsgHistoryAll
}

File: pump/danar/src/main/java/info/nightscout/androidaps/danar/comm/MsgHistoryBolus.kt
package info.nightscout.androidaps.danar.comm

import dagger.android.HasAndroidInjector
import info.nightscout.rx.logging.LTag


class MsgHistoryBolus(
    injector: HasAndroidInjector
) : MsgHistoryAll(injector) {

    init {
        setCommand(0x3101)
        aapsLogger.debug(LTag.PUMPCOMM, "New message")
    }
    // Handle message taken from MsgHistoryAll
}